src/tests/intg/test_ts_cache.py

```diff
@@ -212,12 +212,12 @@ def get_attrs(ldb_conn, type, name, domain, attr_list):
     ts_attrs = dict()
 
     for attr in attr_list:
-        sysdb_attrs[attr] = ldb_conn.get_entry_attr(
-                                     sssd_ldb.CacheType.sysdb,
-                                     type, name, domain, attr)
-        ts_attrs[attr] = ldb_conn.get_entry_attr(
-                                     sssd_ldb.CacheType.timestamps,
-                                     type, name, domain, attr)
+        sysdb_attrs[attr] = \
+            ldb_conn.get_entry_attr(sssd_ldb.CacheType.sysdb,
+                                    type, name, domain, attr)
+        ts_attrs[attr] = \
+            ldb_conn.get_entry_attr(sssd_ldb.CacheType.timestamps,
+                                    type, name, domain, attr)
```

Comment:
Could you explain why it is not a problem with python-pep8-1.6.2-9?

Because I do not like it. I do not now why.

Reply:
It depends whether E126 is disabled by default or not.